evotuning-prototype.py

```diff
@@ -1,30 +1,67 @@
 """Functions for evolutionary tuning."""
 
-from jax_unirep import evotune
-from jax_unirep.evotuning import fit
+# import pandas as pd
+from jax_unirep import evotune, fit
 from jax_unirep.params import add_dense_params
-from jax_unirep.utils import load_params_1900
+from jax_unirep.utils import dump_params, load_params_1900
 
-sequences = ["HASTA", "VISTA", "ALAVA", "LIMED", "HAST"] * 3
-long_seqs = [
-    "AJBGAJHLVSJHLVDJLGDJKGHDKJDGBFGFJKHFBKJHDBDHKJGDHJDGKLGLKJSGSHDGVHJDBDKLJFKJFHDJGDHKJLSGHJSG",
-    "HDKJLFGHLKJFGFKLHJGFDHJDGHJDFGJKDFGDJKDGLHJKDGDJKLHDKJDHJKDGDLHJKGDLJHDKDLDJKDHJDJHGDJKGSDJG",
-    "JDHJKDHJKDHJKDGDKJLGDJHDGDHDGKJDHGKJDLHDKJHFKJLHFJKDJHIDHDIOUDHODHUKDUHDHDUHDKUHDKUDHKDUKKKK",
+# Test sequences:
+sequences = ["HASTA", "VISTA", "ALAVA", "LIMED", "HAST", "HAS", "HASVASTA"] * 5
+holdout_sequences = [
+    "HASTA",
+    "VISTA",
+    "ALAVA",
+    "LIMED",
+    "HAST",
+    "HASVALTA",
 ] * 5
 
-# params = dict()
-# params = add_dense_params(params, "dense", 1900, 25)
-# params["mLSTM1900"] = load_params_1900()
-# params = fit(params=params, sequences=sequences, n=10)
+# To start with Random params instead of mLSTM weights from UniRep:
+"""
+params = dict()
+params = add_dense_params(params, "dense", 1900, 25)
+params["mLSTM1900"] = load_params_1900()
+params = fit(params=params, sequences=sequences, n=10)
+"""
```

Comment:
I think this is outdated, right? Should be removed in that case.

Reply:
Yes, that is right, I didn't catch it. It uses the old API and shouldn't be included as part of the examples.